app/controllers/dashboard_controller.rb

```diff
@@ -26,8 +26,14 @@ def set_draft_works_and_status_map
     @draft_works = []
     @status_map = current_user.your_works.to_h do |work|
       version_status = druid_to_status_map[work.druid] || VersionStatus::NilStatus.new
-      @draft_works << work if version_status.draft? && work.user == current_user
+      @draft_works << work if draft?(work, version_status)
```

Comment:
Before we used to check that the work user was the current user. Is that no longer needed?

Reply:
Oh - good catch, I think we do still need it, I'll restore. it's not needed for the pending check, but it is for the draft check because otherwise works that aren't yours but are in your collections might show up here. I was going back and forth on some refactoring and missed bringing this back.